packages/builder/plugin-swc/package.json

```diff
@@ -64,7 +64,7 @@
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.5",
     "@modern-js/builder-shared": "workspace:*",
-    "@modern-js/swc-plugins": "0.5.5",
+    "@modern-js/swc-plugins": "0.5.7",
```

Comment:
libuild has been moved to this repo, so we can bump the `@modern-js/swc-plugins` of libuild at the same time

Reply:
Done